tests/functional/aws-node-sdk/test/bucket/put.js

```diff
@@ -285,6 +293,21 @@ describe('PUT Bucket - AWS.S3.createBucket', () => {
                     done();
                 });
             });
+
+            it('should return errors InvalidLocationConstraint', done => {
+                bucketUtil.s3.createBucket(
+                    {
+                        Bucket: bucketName,
+                        CreateBucketConfiguration: {
+                            LocationConstraint: 'location-dmf-v1',
+                        },
+                    }, err => {
+                    assert.strictEqual(err.code,
+                    'InvalidLocationConstraint');
+                    assert.strictEqual(err.statusCode, 400);
+                    done();
```

Comment:
```suggestion
                        assert.strictEqual(
                            err.code,
                            'InvalidLocationConstraint',
                        );
                        assert.strictEqual(err.statusCode, 400);
                        done();
```
This is how I would format this function, the first assert blends in and the string on the second like just looks like any other expression. Using one line per argument separates the call from the rest.

Reply:
Done: https://github.com/scality/cloudserver/pull/4532/files#diff-850d88d11c65f5ba1e0ef0f44d7eebc4d07d4965a637625e13f452cc5de5d9e3R310